### PR TITLE
refactor: set gas limit to 2.8M for migrateAll and 2M for all crvAAVE vaults

### PR DESCRIPTION
**Pull request by x48-crypto**

### CODE REVIEW

File: app/containers/Vaults/saga.js [hunk 1]
```diff
@@ -407,6 +432,17 @@ function* migrateVault(action) {
...
+      vaultMigrationData.vaultFrom,
+      vaultMigrationData.vaultTo,
+      {
+        from: account,
+        gas: 2800000,
+      },
+    );
+
     yield call(
       trustedMigratorContract.methods.migrateAll.cacheSend,
```
Comment (xgambitox) [inline app/containers/Vaults/saga.js:447]: should we delete one call? It will try to execute one after the other

Comment (dudesahn) [reply app/containers/Vaults/saga.js:447]: yeah this appears to be setting all vaults to 2800000 gas limit...which should be fine, but means we don't need the second call and should move the error catch up to the first one probably, right?

Comment (x48-crypto) [reply app/containers/Vaults/saga.js:447]: resolved

**Comment by x48-crypto** [comment]
Closed due to merge conflicts

**Pull request closed without merging**